modules/vpc/variables.tf

```diff
@@ -207,3 +207,45 @@ variable "create_nat_gateways_private_b" {
   type        = bool
   default     = false
 }
+
+variable "create_vpc_flow_logs" {
+  description = "Controls if VPC logs should be created for the VPC"
+  type        = bool
+  default     = true
```

Comment:
Shouldn't we default this to false ?

Reply:
@tonynv Should we default it to false and leave it to the users to decide if they want to enable it?

Reply:
Yes

Reply:
Updated #18